### PR TITLE
WIP - Basic Visit Example

### PR DESCRIPTION
**Pull request by vfil**
I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

File: basic-visit-java/src/main/java/ai/vespa/example/visit/Visit.java [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+public class Visit {
+
+    public static void main(String[] args) throws Exception {
+        VisitorParameters params = new VisitorParameters("true");
+        params.setLocalDataHandler(new DumpVisitorDataHandler() {
+
+            @Override
+            public void onDocument(Document doc, long timeStamp) {
+                System.out.print(doc.toXML(""));
```
Comment (kkraune) [inline basic-visit-java/src/main/java/ai/vespa/example/visit/Visit.java]: Vespa Team is deprecating XML interfaces and JSON is more future proof, so toJson is probably better

**Comment by kkraune** [comment]
btw: I am improving the documentation wrt inconsistent buckets these days - I notice you found this problem too :-)

File: basic-visit-java/src/main/java/ai/vespa/example/visit/Visit.java [hunk 2]
```diff
@@ -2,19 +2,19 @@
 
 import com.yahoo.document.Document;
 import com.yahoo.document.DocumentId;
-import com.yahoo.documentapi.DocumentAccess;
-import com.yahoo.documentapi.DumpVisitorDataHandler;
-import com.yahoo.documentapi.ProgressToken;
-import com.yahoo.documentapi.VisitorControlHandler;
-import com.yahoo.documentapi.VisitorParameters;
-import com.yahoo.documentapi.VisitorSession;
+import com.yahoo.documentapi.*;
```
Comment (kkraune) [inline basic-visit-java/src/main/java/ai/vespa/example/visit/Visit.java]: we prefer not using * imports

**Comment by kkraune** [comment]
I guess the log file was not meant to be added

**Comment by vfil** [comment]
@kkraune thank you for the review, I will address your comments. It is still WIP as it seems doesn't work if you run it outside the vespa cluster in another process. Logs are for sharing debug info will delete them.

**Pull request closed without merging**